wordle-web/layouts/default.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <v-app>
+    <v-app-bar color="secondary text-primary">
+      <v-app-bar-title class="font-weight-bold">Pink Wordle</v-app-bar-title>
+      <v-btn to="/">Home</v-btn>
+      <v-btn to="/game">Game</v-btn>
+    </v-app-bar>
+
+    <v-main>
+      <slot />
+    </v-main>
+  </v-app>
+</template>
```

Comment:
Nice layout! I saw this was a thing, but didn't try to implement it. You're pushing me to figure this stuff out!

Reply:
Yes! I watched a video going through Nuxt3 and all of its features and one of them were layouts. It's pretty cool since you can define other layouts if you want to so not all the views used the same `default.vue` layout. Although I do think that maybe its not as useful since we may not need more layouts so putting in it `index.vue` might be better option